code/game/objects/items/devices/radio/cyborg.dm

```diff
@@ -38,17 +38,13 @@
 		else
 			user << "This radio doesn't have any encryption keys!"
 
-	if(istype(W, /obj/item/device/encryptionkey/))
-		if(keyslot)
-			user << "The radio can't hold another key!"
-			return
-
-		if(!keyslot)
+	if (istype(W, /obj/item/device/encryptionkey))
+		if (!isnull(keyslot))
+			user << "<SPAN CLASS='notice'>The radio can't hold another key!</SPAN>"
+		else
 			user.drop_item()
```

Comment:
nooo it will be inside the cyborg instead of the cyborg radio
it uses .loc internally